### DISCUSSION TITLE
Track appearance, return to default

### DISCUSSION THREAD

**Discussion by spyke555** [osmandapp/OsmAnd#11536] [Q&A] [answered]
Hi all,

I am using imported GPX tracks that use the gpxx:DisplayColor extension to set the colour of the tracks. When I open these GPX files in OsmAnd the tracks are coloured as per the GPX file contents which is good.

But then I made the mistake of setting a colour for the tracks in the Configure Maps > Tracks > Appearance menu and all my tracks became the same colour. To restore proper operation I had to remove all my tracks from OsmAnd and then re-import them all.

Just wondering if there is a way to clear the appearance setting to return the tracks to the values from the GPX another way? Or is delete and re-import the only way?

Thanks for an awesome app!

**Comment by vshcherb** [comment, marked as answer]
This will be fixed in upcoming release: both how we change color and how the dialog works. So now files in Tracks folder will stay original (if you changed only the color), so if you want to get original you can just take it from file system. Unfortunately there is no button "Restore original" yet, though it would be confusing a bit cause not everything is revertable.
